src/components/EarnDashboard/components/PositionDetails/StakingPositionsByProvider.tsx

```diff
@@ -215,9 +214,22 @@ export const StakingPositionsByProvider: React.FC<StakingPositionsByProviderProp
                 .div(bn(10).pow(assets[opportunityAssetId]?.precision ?? 18))
                 .toFixed()
 
+          const fiatRewardsAmount = calculateRewardFiatAmount({
+            rewardAssetIds: row.original.rewardAssetIds,
+            rewardsCryptoBaseUnit: row.original.rewardsCryptoBaseUnit,
+            assets,
+            marketDataUserCurrency,
+          })
+
+          const hasValue =
+            bnOrZero(opportunity.fiatAmount).gt(0) || bnOrZero(fiatRewardsAmount).gt(0)
+
+          // Note, this already includes rewards. Let's not double-count them
+          const totalFiatAmount = bnOrZero(row.original.fiatAmount).toFixed(2)
```

Comment:
blocking: crypto amount should also include rewards else discrepancy

Reply:
Update: We most likely don't want that, see https://github.com/shapeshift/web/pull/9286/files#r2033206229 for rationale